components/match2/wikis/splitgate/brkts_wiki_specific.lua

```diff
@@ -0,0 +1,49 @@
+---
+-- @Liquipedia
+-- wiki=splitgate
+-- page=Module:Brkts/WikiSpecific
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local FnUtil = require('Module:FnUtil')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local _EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
+
+local WikiSpecific = Table.copy(require('Module:Brkts/WikiSpecific/Base'))
+
+WikiSpecific.processMatch = FnUtil.lazilyDefineFunction(function()
+	local InputModule = Lua.import('Module:MatchGroup/Input/Custom', {requireDevIfEnabled = true})
+	return InputModule.processMatch
+end)
+
+WikiSpecific.processMap = FnUtil.lazilyDefineFunction(function()
+	local InputModule = Lua.import('Module:MatchGroup/Input/Custom', {requireDevIfEnabled = true})
+	return InputModule.processMap
+end)
+
+WikiSpecific.processOpponent = FnUtil.lazilyDefineFunction(function()
+	local InputModule = Lua.import('Module:MatchGroup/Input/Custom', {requireDevIfEnabled = true})
+	return InputModule.processOpponent
+end)
+
+WikiSpecific.processPlayer = FnUtil.lazilyDefineFunction(function()
+	local InputModule = Lua.import('Module:MatchGroup/Input/Custom', {requireDevIfEnabled = true})
+	return InputModule.processPlayer
+end)
```

Comment:
General thought rather than a something review specific. Should we move this into /Base? Most wikis won't need to override it and those that need can still override it.
CC: @warnull

Reply:
imo a good idea, but should be done via a sep. PR

Reply:
PR #737 has been opened